modeshape-parent/pom.xml

```diff
@@ -1271,6 +1271,12 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
```

Comment:
`modeshape-parent` already defines the http-client dependency:
```
<dependency>
  <groupId>org.apache.httpcomponents</groupId>
   <artifactId>httpclient</artifactId>
   <version>${version.org.apache.httpcomponents}</version>
 </dependency>
```

This change should use that version of the library.